package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-native-safari-view",
   "version": "0.1.0",
   "description": "A React Native wrapper for Safari View Controller",
-  "main": "SafariViewMananger.ios.js",
+  "main": "index.ios.js",
   "scripts": {
```

Comment:
Oh sorry, I missed that. 😁

I'd still like to wait for the outcome of facebook/react-native#1808 prior to renaming the ios.js file. 

I'd rather the file names remain consistent with the way `react-native new-library` structures libraries.